### DISCUSSION TITLE
Configure Windows Path Separator to Forward Slash

### DISCUSSION THREAD

**Discussion by JeremyBYU** [nushell/nushell#9237] [Q&A] [answered]
Hi all,
I have been trying out nushell (Windows 11) and have enjoyed many of its features. However, I was wondering if there was any way to adjust that path separator used in nushell to use forward slashes. I have grown accustomed to them when using git bash.  I see that it transparently converts a forward slash to backslash when typing. However, I would like it to remain in forward slash and be interpreted correctly.

NuShell:

```
~> cd \Users\Jerem\Documents\                                                                     05/19/2023 07:31:00 AM
~/Documents>    
```

Git bash:
```
❯ cd /c/Users/Jerem/Documents/
```

Thanks!

**Comment by fdncred** [comment, marked as answer]
You can just change your left prompt to replace backslashes with forward slash. I do that in my nu_scripts/modules/prompts/oh-my.nu script.

**Reply by LovesTha** [reply]
I know this is an ancient post, but any clues on how to do this? Searching gives me unrelated pages and this thread. :(

**Reply by fdncred** [reply]
There's no built-in way to change all backslashes to slashes right now. However, there has been some talk on Discord about it recently. Someone offered to make a PR for it. We'll see if it happens.
